src/main/java/me/zeroeightsix/kami/util/Note.java

```diff
@@ -0,0 +1,48 @@
+package me.zeroeightsix.kami.util;
+
+public class Note {
+    int note;
+    int track;
+
+    public Note(int note, int track) {
+        this.note = note;
+        this.track = track;
+    }
+
+    public int getNote() {
+        return note;
+    }
+
+    public int getNotebotNote() {
+        return getNotebotKey(note);
+    }
+
+    public int getTrack() {
+        return track;
+    }
+
+    @Override
+    public String toString() {
+        return getKey(note) + "[" + track + "]";
+    }
+
+    public static String getKey(int note) {
+        String[] keys = new String[]{
+                "F#", "G", "G#", "A", "A#", "B", "C", "C#", "D", "D#", "E", "F",
+                "F#2", "G2", "G#2", "A2", "A#2", "B2", "C2", "C#2", "D2", "D#2", "E2", "F2",
+                "F#3"
+        };
```

Comment:
This should be a static field because it is always the same. No need to create new array everytime